setup.py

```diff
@@ -24,7 +24,6 @@
       include_package_data=True,
       install_requires=[
                         "click>=7.1.1",
-                        "pypsa",
```

Comment:
Is there a reason why pypsa is not on the setup.py ? It's mandatory to use the package, no ? (beside, pypsa license is now ok)